src/InteractiveAtom.test.tsx

```diff
@@ -6,8 +6,13 @@ import { InteractiveAtom } from './InteractiveAtom';
 
 describe('InteractiveAtom', () => {
     it('should render', () => {
-        const { getByText } = render(<InteractiveAtom id="123abc" />);
+        const { getByText } = render(
+            <InteractiveAtom
+                id="123abc"
+                url="https://api.nextgen.guardianapps.co.uk/embed/atom/interactive/interactives%2F2020%2F03%2Fcovid-19-uk%2Fdefault"
```

Comment:
this would be more just testing `unifyPageContent.tsx` than testing `InteractiveAtom`which I am happy to do. I agree though that an integration test makes more sense here